v2/Makefile

```diff
@@ -11,7 +11,10 @@ i18n_lua = lua/openresty_org/i18n.lua
 templates_lua = lua/openresty_org/templates.lua
 
 lua_files := $(sort $(filter-out $(i18n_lua) $(templates_lua),$(wildcard lua/*.lua lua/openresty_org/*.lua)))
-tt2_files := $(sort $(wildcard templates/*.tt2))
+tt2_files := $(sort $(wildcard templates/*.tt2)) templates/posts-slide-*.tt2
```

Comment:
This is wrong. `templates/posts-slide-*.tt2` might not exit for a fresh clone. They are generated files anyway. You should always write their names fully and explicitly.

Reply:
changed.